tests/modules/missions/resources/test_create_mission.py

```diff
@@ -18,13 +19,17 @@ def test_create_and_delete_mission(flask_app_client, data_manager_1):
         MissionUserAssignment,
     )
 
+    nonce = random_nonce(8)
+    title = 'This is a test mission (%s), please ignore' % (nonce,)
```

Comment:
This seems to be duplicated in most of the missions tests. I'd suggest a missions util of get_random_title.

Reply:
I've added a new utility for `mission_utils.make_name('mission')`, which will remove some of this code duplication